src/components/OSCALBackMatter.js

```diff
@@ -91,7 +91,12 @@ export default function OSCALBackMatter(props) {
 
   const getMediaType = (rlink) =>
     rlink["media-type"] ||
-    guessExtensionFromHref(getAbsoluteUrl(rlink.href, props.parentUrl));
+    guessExtensionFromHref(
+      getAbsoluteUrl(
+        rlink.href,
+        props.parentUrl.startsWith(".") ? null : props.parentUrl
```

Comment:
This too

Reply:
Without using the absolute url `guessExtensionFromHref` fails to parse the type of `diagram.png`. `diagram.png` is not an absolute url, so the type is always `Unknown`.

https://github.com/EasyDynamics/oscal-react-library/blob/5b4a79269232343deb0c53ce7498d450063b0fae/src/components/oscal-utils/OSCALLinkUtils.js#L57-L70